packages/odyssey-react-labs/src/datePickerTheme.tsx

```diff
@@ -61,12 +62,12 @@ const todayDotStyles: StateStyles = {
   default: ({ theme }) => ({
     backgroundColor: theme.palette.primary.main,
     borderRadius: "50%",
+    bottom: theme.spacing(1),
     content: '" "',
     height: `${(2 / 16) * (16 / 14)}rem`,
     position: "absolute",
-    bottom: theme.spacing(1),
-    width: `${(2 / 16) * (16 / 14)}rem`,
     transform: "translateY(-50%)",
+    width: `${(2 / 16) * (16 / 14)}rem`,
```

Comment:
These won't hold if someone overrides `typography.fontSize`. What's the goal here? Maybe I can point you toward something safer.

Reply:
For ex, we use `em` for our Status dots, so that they're always related to the size of the Status type:

```
"&::before": {
  content: "''",
  width: ".57em",
  height: ".57em",
  marginInlineEnd: theme.spacing(2),
  borderRadius: "100%",
  backgroundColor: theme.palette.text.secondary,
}
```

(We should probably have variables for these dot sizes, but you get the idea.)

Reply:
If someone overrides `typography.fontSize`, you're saying it won't scale up the `width`? Yeah, I'm assuming fonts sale with the HTML which is why I'm using `rem`. But if they don't scale, then this `2px` won't be consistent.

But I agree, they should be set as token variables, not `2px` as `rem`.

Reply:
Updated to this:
```suggestion
    width: '0.14285714285714285em', // 2px
```